### DISCUSSION TITLE
Incorrect labeling of channel names

### DISCUSSION THREAD

**Discussion by Will760** [BodenmillerGroup/steinbock#178] [Q&A] [answered]
Hi All,

I have some IMC data and would like to use Steinbock to conduct the pre-processing for. My issue is that some of the images have discrepancies in their channel names. For instance, one channel might be named 'CD8' and another 'cd8' or 'CD8a'.

I understand I can change the .txt files but I don't see any way to change the .mcd files to standardize the channel names. Is there anyway I can change the names in the .mcd files? or is there some alteration I can make to the 'steinbock preprocess imc images --unzip --hpf 50' command to use .txt files instead of .mcd files? What is the work-around here?

Apologies if this question is a bit simple, I am very new to the computational side of IMC so am slowly finding my way.

Thanks for your help!

Will

**Comment by nilseling** [comment]
Hi @Will760 

how come the images have different channel names? Also, the important thing is that the metal tags (channels) are the same across all images that you want to analyse in one go. The channel name can be adjusted in the panel file.

**Reply by Will760** [reply]
Hi @nilseling ,

Thanks for your super speedy response. Sorry, that was me not being clear, I meant image labels. Occasionally with different members of the team taking images, the pre-set templates were ignored and the labels were incorrectly transcribed. The channels are of course the same throughout.

Sorry for the confusion.

**Reply by nilseling** [reply, marked as answer]
Hmm, then it shouldn't be a problem to use `steinbock` for processing. You will just need to generate a consistent panel file that can be applied to all images.
